src/phoenix/experimental/evals/models/anthropic.py

```diff
@@ -157,7 +157,10 @@ async def _async_generate(self, prompt: str, **kwargs: Dict[str, Any]) -> str:
 
         return str(response)
 
-    async def _async_generate_with_retry(self, **kwargs: Any) -> Any:
+    async def _async_generate_with_retry(self, **kwargs: Any) -> Any:        
+        # instruction is an invalid input to Anthropic models, it is passed in by
+        # BaseEvalModel.__call__ and needs to be removed
+        kwargs.pop("instruction", None)
```

Comment:
@anticorrelator can you just pull this part out and open a PR without the rest? This branch is stale due to it modifying non-existing retry methods

Reply:
@mikeldking this line is already in our code: https://github.com/Arize-ai/phoenix/blob/main/src/phoenix/experimental/evals/models/anthropic.py#L150